core/domain/prod_validation_jobs_one_off.py

```diff
@@ -110,67 +351,3183 @@ def validate(cls, item):
+                if isinstance(summary_model_output_value, list):
+                    summary_model_output_value = (',').join(
+                        summary_model_output_value)
+
+                collection_model_output_value = (
+                    value_in_collection_model)
+                if isinstance(collection_model_output_value, list):
+                    collection_model_output_value = (',').join(
+                        collection_model_output_value)
+
+                if value_in_summary_model != value_in_collection_model:
+                    cls.errors['%s field check' % property_name].append((
+                        'Entity id %s: %s field in entity: %s does not match '
+                        'corresponding collection %s field: %s') % (
+                            item.id, property_name, summary_model_output_value,
+                            collection_model_property_name,
+                            collection_model_output_value))
+
+        collection_rights_model_properties_dict = {
+            'status': 'status',
+            'community_owned': 'community_owned',
+            'owner_ids': 'owner_ids',
+            'editor_ids': 'editor_ids',
+            'viewer_ids': 'viewer_ids',
+        }
+
+        for (_, collection_rights_model) in collection_rights_model_tuples_list:
+            for (property_name, collection_rights_model_property_name) in (
+                    collection_rights_model_properties_dict.iteritems()):
+                value_in_summary_model = getattr(item, property_name)
+                value_in_collection_rights_model = getattr(
+                    collection_rights_model,
+                    collection_rights_model_property_name)
+
+                summary_model_output_value = value_in_summary_model
+                if isinstance(summary_model_output_value, list):
+                    summary_model_output_value = (',').join(
+                        summary_model_output_value)
+
+                collection_rights_model_output_value = (
+                    value_in_collection_rights_model)
+                if isinstance(collection_rights_model_output_value, list):
+                    collection_rights_model_output_value = (',').join(
+                        collection_rights_model_output_value)
+
+                if value_in_summary_model != value_in_collection_rights_model:
+                    cls.errors['%s field check' % property_name].append((
+                        'Entity id %s: %s field in entity: %s does not match '
+                        'corresponding collection rights %s field: %s') % (
+                            item.id, property_name, summary_model_output_value,
+                            collection_rights_model_property_name,
+                            collection_rights_model_output_value))
+
+    @classmethod
+    def _get_custom_validation_functions(cls):
+        return [
+            cls._validate_language_code, cls._validate_node_count,
```

Comment:
(Optional nit) OK to do one function per line here. Easier to read and reinforces the idea that these functions are called one-by-one in order. Ditto elsewhere.

Reply:
Done.